src/components/layout/SwapLayout/Header/Navigation.styled.ts

```diff
@@ -22,10 +22,9 @@ export const NavLinksWrapper = styled.div<{ $open?: boolean; $responsive: boolea
   a:visited {
     border: 0.125rem solid transparent;
     color: var(--color-text-secondary);
-    font-weight: bolder;
     text-decoration: none;
     transition: color 0.2s ease-in-out, background 0.2s ease-in-out;
-    font-weight: var(--font-weight-bold);
+    font-weight: var(--font-weight-bold, bolder);
```

Comment:
??

Reply:
linter wanted a default in case `--font-weight-bold` isn't available

Reply:
cool, didn't know this worked